nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -65,6 +65,9 @@ def bert_extended_attention_mask(attention_mask):
     # [b, 1, s, s]
     extended_attention_mask = attention_mask_bss.unsqueeze(1)
 
+    # HF Masking is equivalent to the one below
+    # extended_attention_mask = (attention_mask.unsqueeze(1) * torch.ones_like(attention_mask).unsqueeze(2)).unsqueeze(1)
```

Comment:
can you comment this?

Reply:
I didn't work on these files. These are pulled from the latest main branch. If you want, I can revert these files to r1.23.0.

Reply:
I have reverted these changes to r1.23.0.